frontend/test/metabase-db/postgres/custom-column.cy.spec.js

```diff
@@ -0,0 +1,58 @@
+import {
+  signInAsAdmin,
+  restore,
+  addPostgresDatabase,
+  popover,
+} from "__support__/cypress";
+
+describe("postgres > question > custom columns", () => {
+  before(() => {
+    restore();
+    signInAsAdmin();
+    addPostgresDatabase();
```

Comment:
@dacort is there a way to use this as part of the default data set instead of adding it manually every time before tests?